### ISSUE TITLE
Can't Horizontal Scroll on Wide Boards

### ISSUE THREAD

**Issue by nickian** [greyaz/ThemeRevision#28]
I have updated to the newest version of Kanboard and the newest release of the plugin. Now I can no longer horizontally scroll on boards that are wider than my screen. When I remove the Theme/Plugin, horizontal scrolling comes back and works normal.

**Comment by nickian** [comment]
This is strange. I have two Kanboard installations using this theme. This is only happening on one. All the specs are the same. I removed all custom CSS and plugins, I reinstalled ThemeRevision. It still happens on this installtion. Meanwhile, it's not happening on my other one. Any idea what could be causing this? The default theme is allowing horizontal scrolling, just not ThemeRevision.

**Comment by greyaz** [comment]
The board view provides a "horizontal scrolling" option by clicking the gear icon in the top-left corner. Have you tried it?

**Comment by nickian** [comment]
Yes, toggling between horizontal scrolling and compact view doesn't seem to change anything for me.

**Comment by greyaz** [comment]
@nickian I met this issue after upgrading to Kanboard 1.2.39.
The following commit led to this issue: https://github.com/kanboard/kanboard/pull/5523/commits/06ffe5a13628331b88e5723e59a37c26196ad947
You can set a CSS style to fix it temporarily:
```
body{
overflow-x: unset;
}
```

**Comment by nickian** [comment]
Yes, that fix works. Thanks!

**Comment by wucherpfennig** [comment]
```css
#board-container{
  overflow-x: auto !important;
  width: 100%;
}
```

**Comment by Heart1010** [comment]
@wucherpfennig Thanks 👍
